src/charm.py

```diff
@@ -815,6 +815,8 @@ def _install_snap_packages(self, packages: List[str]) -> None:
                     snap_package.ensure(
```

Comment:
Good opportunity to remove the `if not snap_package.present` check, and just leave `snap_package.ensure() & hold`

Reply:
🙇